package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
+    "heroku-prebuild": "cd client && npm i",
```

Comment:
I recommend caching the client node_modules too on Heroku:

Add this to your package.json.
```
"cacheDirectories": [
    "client/node_modules",
    "node_modules"
  ]
```

See https://devcenter.heroku.com/articles/nodejs-support#custom-caching